src/Compilers/Core/Portable/SourceFileResolver.cs

```diff
@@ -62,9 +62,9 @@ public SourceFileResolver(ImmutableArray<string> searchPaths, string baseDirecto
                     }
 
                     var value = kv.Value;
-                    if (value == null || value.Length == 0)
+                    if (value == null)
                     {
-                        throw new ArgumentException(CodeAnalysisResources.EmptyValueInPathMap, nameof(pathMap));
```

Comment:
This file is automatically generated and shouldn't be modified manually, the next build will simply undo all the changes you make. The .resx file should be modified instead.